overlord/configstate/configcore/certs.go

```diff
@@ -32,6 +33,221 @@ import (
 	"github.com/snapcore/snapd/overlord/configstate/config"
 )
 
+type certificate struct {
+	Name     string
+	Path     string
+	RealPath string
+}
+
+// This function validates the name of the certificate. We only allow certificates
+// with the correct suffix ".crt" except for the ca-certificates.crt,
+// or if the name is in the blockedCerts lists.
+func isBlocked(cert certificate, blockedCerts []string) bool {
```

Comment:
What's the idea with blockedCerts? Are we going to take `/etc/ca-certificates.conf` file into account to block disabled certificates (those with `!` in front)?  SIlly question: do we need to look at actual contents of disabled certs so that a snap cannot try to import a known compromised certificate under a different file name?

Reply:
Yes exactly! This is what the point is of the blockedCerts list. It's so we are able to imitate the '!' functionality of update-ca-certificates. 

This is a really good question. It raises some good concerns. @mvo5 What is your thoughts here?

Reply:
I think we will actually block based on fingerprint so this function might not be exactly be what we want in the end